pages/en/get-involved/index.md

```diff
@@ -18,7 +18,8 @@ layout: contribute.hbs
 
 ## Learning
 
-- [Official API reference documentation](https://nodejs.org/api/) details the Node.js API.
+- [Official Learn section](/learn)
+- [Official API reference documentation](/api/) details the Node.js API.
```

Comment:
/api is an external page. It should use absolute URLs.

Reply:
On that topic, if you can make a scan and search any relative URL that should be absolute (ie /downloads (not /download), /docs, /api, /dist. /metrics) they should be absolute URLs